pkg/utils/cgroups.go

```diff
@@ -80,7 +80,7 @@ func isProcess(processID string) bool {
 	return true
 }
 
-func getTasksInContainer(cgroupParentDir, containerID string, onlyProcesses bool) ([]string, error) {
+func getTasksInContainer(cgroupParentDir, containerID string, podID string, onlyProcesses bool) ([]string, error) {
```

Comment:
nitpick: it could be a bit more consistent with the rest of the functions if we swapped the `containerID, podID` arguments the other way around to `podID, containerID`.